extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -273,6 +308,18 @@ object PrivilegesBuilder {
           inputObjs += databasePrivileges(db.get)
         }
 
+      case "CreateNamespace" =>
+        // for Spark 3.1 and below
+        val namespace = getFieldValOption[Seq[String]](plan, "namespace")
```

Comment:
use the spark version as condition instead of the current way, FYI, SetCatalogAndNamespace as an example

Reply:
applied.